packages/legend-application-studio/src/components/editor/side-bar/testable/GlobalTestRunner.tsx

```diff
@@ -373,6 +383,31 @@ export const GlobalTestRunner = observer(
     const editorStore = useEditorStore();
     const globalTestRunnerState = props.globalTestRunnerState;
     const isDispatchingAction = globalTestRunnerState.isDispatchingAction;
+    const sdlcState = editorStore.sdlcState;
+    const currentWorkspace = sdlcState.currentWorkspace;
+    const plugins = editorStore.pluginManager.getApplicationPlugins();
+    const testRunnerTabs = (Object.values(GLOBAL_TEST_RUNNER_TABS) as string[])
+      .concat(
+        plugins.flatMap(
+          (plugin) =>
+            (
+              plugin as STO_ProjectOverview_LegendStudioApplicationPlugin_Extension
+            ).getExtraTestRunnerTabsClassifiers?.() ?? [],
+        ),
+      )
+      .map((e) => ({
+        value: e,
+        label: prettyCONSTName(e),
+      }));
+
+    const [selectedTab, setSelectedTab] = useState(
+      GLOBAL_TEST_RUNNER_TABS.GLOBAL_TEST_RUNNER.valueOf(),
```

Comment:
why is `valueOf` needed here ?

Reply:
I think to get string value, since `const changeTab = (tab: string): void => {` the change function type is string and also the `getExtraTestRunnerTabsClassifiers` gives strings for extension mechanism